kimchi/src/tests/keccak.rs

```diff
@@ -0,0 +1,60 @@
+use std::array;
+
+use crate::circuits::{
+    constraints::ConstraintSystem,
+    gate::CircuitGate,
+    polynomials::keccak::{self, ROT_TAB},
+    wires::Wire,
+};
+use ark_ec::AffineCurve;
+use mina_curves::pasta::{Fp, Pallas, Vesta};
+use rand::Rng;
+
+//use super::framework::TestFramework;
+type PallasField = <Pallas as AffineCurve>::BaseField;
+
+fn create_test_constraint_system() -> ConstraintSystem<Fp> {
+    let (mut next_row, mut gates) = { CircuitGate::<Fp>::create_keccak(0) };
+
+    // Temporary workaround for lookup-table/domain-size issue
```

Comment:
I wish we don't need this anymore.

Reply:
desire or affirmation?

Reply:
desire

Reply:
I wish proof-system automatically took care of this optimally